crates/next-core/src/pages_structure.rs

```diff
@@ -117,33 +117,29 @@ pub async fn find_pages_structure(
     let pages_root = project_root
         .join("pages".into())
         .realpath()
-        .resolve()
+        .to_resolved()
         .await?;
-    let pages_root = Vc::<FileSystemPathOption>::cell(
```

Comment:
This code was a bit whacky. It was calling `Vc::cell(...).resolve().await`, which doesn't make much sense because a newly constructed `Vc` is always resolved...

Cleaned this up a bit since I was here, but this isn't something I'd expect a codemod to do.

I wasn't able to find this antipattern anywhere else in our codebase (checked with astgrep).